Majasdarbs/03-majasdarbs/majasdarbs 3/style.css

```diff
@@ -0,0 +1,54 @@
+h1 {
+
+  text-shadow: -1px 0 black, 0 1px black, 1px 0 black, 0 -1px black;
+  font-family: 'Inter', sans-serif;
```

Comment:
font-family should be specified either on body or html, so it is used for all elements, instead of specifying separately for each selector.

Reply:
Also 99% of the times you use only 1 font-family, but for playing around its fine.